statsmodels/stats/sandwich_covariance.py

```diff
@@ -533,15 +601,22 @@ def cov_cluster(results, group, use_correction=True):
     nobs, k_params = xu.shape
     n_groups = len(clusters) #replace with stored group attributes if available
 
-    cov_c = _HCCM2(hessian_inv, scale)
+    if jackknife == False:
+        cov_c = _HCCM2(hessian_inv, scale)
```

Comment:
AFAICS, `scale` above is not used in jackknife.
In that case the `scale` computation can be put under this `if` branch